torch/lib/c10d/ProcessGroupMPI.hpp

```diff
@@ -75,13 +75,26 @@ struct WorkEntry {
 class ProcessGroupMPI : public ProcessGroup {
  public:
   class WorkMPI : public ProcessGroup::Work {
+   public:
+    WorkMPI(
+        const char* profilingTitle = nullptr,
+        const c10::optional<std::vector<at::Tensor>>& inputTensors =
+            c10::nullopt)
+        : ProcessGroup::Work(
+              -1,
+              OpType::UNKNOWN,
+              profilingTitle,
+              inputTensors) {}
+
    protected:
     friend class ProcessGroupMPI;
   };
 
   class AsyncWork : public ProcessGroup::Work {
    public:
-    AsyncWork(at::Tensor tensor, MPI_Request request);
+    AsyncWork(at::Tensor tensor, MPI_Request request,
+    const char * profilingTitle = nullptr,
```

Comment:
is the indention correct?

Reply:
It passed lint, but yeah, I think it should be one level in